RELEASE_NOTES.md

```diff
@@ -3,6 +3,8 @@
 ## 2.0.6
 ### Bugfixes
 - Fixed a bug where running systemd in a container on a cgroups v1 system would fail.
+- Fixed a bug where `/etc/passwd` could be re-created every time a container is restarted if the container's `/etc/passwd` did not contain an entry for the user the container was started as.
+- Fixed a bug where containers without an `/etc/passwd` file specifying a non-root user would not start.
 
```

Comment:
I think you should add that `--remote` now works correctly.

Reply:
Done